jbmc/src/java_bytecode/select_pointer_type.h

```diff
@@ -36,6 +36,16 @@ class select_pointer_typet
 
 public:
   virtual ~select_pointer_typet() = default;
+
+  /// Select what type should be used for a given pointer type. For the base
+  /// class we just use the supplied type. Derived classes can override this
+  /// behaviour to provide more sophisticated type selection. Generic
+  /// parameters are replaced with their concrete type.
```

Comment:
:pick: "concrete type" -> "given/specified type argument"? To avoid confusion as "concrete" can mean the opposite of "abstract".
:frog:

Reply:
Good point, I replaced it with `specialized type`